File: cms_server/cms_server/Entities/ConsultProcEntity.cs
using System;

namespace cms_server.Entities
{
    public class ConsultProcEntity
    {
        public int? cr_proc_pk { get; set; }
        public string consult_req_pk { get; set; }
        public string proc_no { get; set; }
        public string proc_desc { get; set; }
        public decimal? reg_price { get; set; }
        public string notes { get; set; }
        public string is_active { get; set; }
        public DateTime? encoded_at { get; set; }
        public string encoder_pk { get; set; }
    }
}

File: cms_web/src/Services/Entities/ConsultProcEntity.ts
interface ConsultProcEntity {
  cr_proc_pk?: number;
  consult_req_pk?: string;
  proc_no?: string;
  proc_desc?: string;
  reg_price?: number;
  notes?: string;
  is_active?: "y" | "n";
  encoded_at?: string | Date;
  encoder_pk?: string;
}

export interface ConsultProcTableModel {
  limit: number;
  count: number;
  begin: number;
  table: Array<ConsultProcEntity>;
}

export default ConsultProcEntity;
